docs/source/checks/nlp/data_integrity/plot_text_property_outliers.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+"""
+.. _nlp__text_property_outliers:
+
+Text Property Outliers
+=======================
+
+This notebooks provides an overview for using and understanding the text property
+outliers check, used to detect outliers in simple text properties in a dataset.
+
+**Structure:**
+
+* `Why Check for Outliers? <#why-check-for-outliers>`__
+* `How Does the Check Work? <#how-does-the-check-work>`__
+* `Which Text Properties Are Used? <#which-text-properties-are-used>`__
+* `Run the Check <#run-the-check>`__
+
+
+Why Check for Outliers?
+-----------------------
+Examining outliers may help you gain insights that you couldn't have reached from taking an aggregate look or by
+inspecting random samples. For example, it may help you understand you have some corrupt samples (e.g.
+texts without spaces between words), or samples you didn't expect to have (e.g. texts in Norwegian instead of English).
+In some cases, these outliers may help debug some performance discrepancies (the model can be excused for failing on
+a totally blank text). In more extreme cases, the outlier samples may indicate the presence of samples interfering with
+the model's training by teaching the model to fit "irrelevant" samples.
+
+
+How Does the Check Work?
+------------------------
+Ideally we would like to directly find text samples which are outliers, but this is computationally expensive and does not
```

Comment:
Don't know if we need all this explanation here. Can just say that this check is about samples that are outliers in the sense that they are extreme / anomalous in one of the pre-ordained set of properties.

Reply:
I disagree... There's no other place where we can explain our logic, and I think this is actually important information